src/ui/components/filters/filteroptionscomponent.js

```diff
@@ -271,6 +281,22 @@ export default class FilterOptionsComponent extends Component {
     if (this.config.storeOnChange) {
       this.apply(this.config.isDynamic);
     }
+
+    if (!this.config.isDynamic) {
+      this.core.globalStorage.on('update', this.name, (data) => {
```

Comment:
Can this listener get fired outside the back nav case? It really gets triggered whenever `resetListener` is called, right? Can that listener be fired for other kinds of events?

Reply:
It definitely _can_
Does it? It does not. To double check I went through a few interactions with a `console.log` in the listener and found it only logged on back/fwd nav.

This is because we normally work with `static-filter-node.FilterBox.filter0` in global storage when working with static filters. The reason we do not update that `static-filter-node` key here is because we would need to
* Find whether or not that query param was a static vs dynamic filter so we can prepend the correct `x-filter-node` prefix.
* Cast the query parameter into a FilterNode
* Update the key with the FilterNode and then update options based on this an update on this key. However, because this key is updated in other places in filteroptions, it would end up being a bigger refactor of the FilterOptions general workflow to make sure we are only updating options for the back nav case.